HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -258,5 +269,6 @@ def customizeHLTforCMSSW(process, menuType="GRun"):
  
     process = customizeHLTfor38761(process)
     process = customizeHLTfor40264(process)
+    process = customizeHLTfor40334(process)
```

Comment:
This customisation is not necessary for the menus in CMSSW. It can become necessary for user menus extracted from `ConfDB`, if those somehow contain a `PSMonitor`.